components/brave_rewards/resources/extension/brave_rewards/content_scripts/github.ts

```diff
@@ -0,0 +1,380 @@
+    let braveTipActionCount = tipElem.shadowRoot.querySelectorAll('.' + tipActionCountClass)[0] as HTMLElement
+    iconContainer.style.paddingBottom = '5px'
+    braveTipActionCount.style.paddingBottom = '2px'
+    tipElem.style.marginRight = '2px'
+  }
+  const children = parent.childNodes
+  if (children.length > 1) {
+    const end = children[children.length - 2]
+    parent.insertBefore(tipElem, end)
+  }
+}
+
+const getReviewItemMetaData = (elem: Element): RewardsTip.MediaMetaData | null => {
+  const ancestor = elem.closest('.discussion-item')
+  if (ancestor) {
+    const authorCollection = ancestor.getElementsByClassName('author')
+    if (authorCollection.length) {
+      const author = authorCollection[0] as HTMLElement
+      const userName = author.innerHTML
+      return {
+        mediaType: 'github',
+        userName: userName || ''
+      }
+    }
+  }
+  return null
+}
+
+const reviewItemInsertFunction = (parent: Element) => {
+  const tipLocation = parent.getElementsByClassName('discussion-item-copy')[0]
+  if (tipLocation) {
+    const tipElem = createBraveTipAction(tipLocation, getReviewItemMetaData)
+    if (tipElem.shadowRoot) {
+      const iconContainerCollection = tipElem.shadowRoot.querySelectorAll('.' + tipIconContainerClass)
+      const braveTipActionCountCollection = tipElem.shadowRoot.querySelectorAll('.' + tipActionCountClass)
+      if (iconContainerCollection.length && braveTipActionCountCollection.length) {
+        const iconContainer = iconContainerCollection[0] as HTMLElement
+        const braveTipActionCount = braveTipActionCountCollection[0] as HTMLElement
+        iconContainer.style.paddingBottom = '5px'
+        braveTipActionCount.style.paddingBottom = '2px'
+      }
+    }
+    tipLocation.appendChild(tipElem)
+  }
+}
+
+const getCommitLinksMetaData = (elem: Element): RewardsTip.MediaMetaData | null => {
+  const ancestor = elem.closest('.commit')
+  if (ancestor) {
+    const avatarCollection = ancestor.getElementsByClassName('avatar')
+    if (avatarCollection.length) {
+      const avatar = avatarCollection[0] as HTMLElement
+      const avatarChildren = avatar.children
+      if (avatarChildren.length) {
+        const userName = (avatarChildren[0] as HTMLImageElement).alt.slice(1)
+        return {
+          mediaType: 'github',
+          userName: userName || ''
+        }
+      }
+    }
+  }
+  return null
+}
+
+const commitLinksInsertFunction = (parent: Element) => {
+  const tipElem = createBraveTipAction(parent, getCommitLinksMetaData)
+  tipElem.style.marginLeft = '9px'
+  parent.appendChild(tipElem)
+}
+
+const getStarringContainerMetaData = (elem: Element): RewardsTip.MediaMetaData | null => {
+  const ancestor = elem.closest('.d-block')
+  if (ancestor) {
+    const aTags = ancestor.getElementsByTagName('A')
+    if (aTags.length) {
+      const aTag = aTags[0]
+      const userName = (aTag as HTMLAnchorElement).href.split('/')[3]
```

Comment:
check if href is undefined and if you have array that has 4 items inside

Reply:
fixed